application-analytics-api/src/main/java/com/xwiki/analytics/configuration/AnalyticsConfiguration.java

```diff
@@ -51,4 +51,9 @@ public interface AnalyticsConfiguration
      * @return the tracking code for Matomo.
      */
     String getTrackingCode();
+
+    /**
+     * @return true if the tracking is enabled, false otherwise.
```

Comment:
you could add a @ since tag, since it's a feature introduced in a later version of the app. `isEnabled` would also fit better with the java naming standards.